app/Jobs/SendReportbackToPhoenix.php

```diff
@@ -33,14 +33,16 @@ public function handle()
     {
         $phoenix = new Phoenix;
 
+        $reportbackItem = $this->reportback->items()->orderBy('created_at', 'desc')->first();
```

Comment:
I guess I'm wondering if there would ever be an instance when we would want all items returned so phoenix could do whatever it wants with all the items after something has been updated. But that is not a blocker for merge, we could always figure that out if it comes up.